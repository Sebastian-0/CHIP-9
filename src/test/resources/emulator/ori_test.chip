
ldi A, 0x0

# Zero flag
ori 0x0
sout A
sout FLAGS

# No flag
ori 0x7C
sout A
sout FLAGS

# Negative flag
ori 0x80
sout A
sout FLAGS

hcf