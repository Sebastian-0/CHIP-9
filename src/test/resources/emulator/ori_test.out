0
128

0x7C
0

0xFC
64